style.css

```diff
@@ -39,10 +51,55 @@ header .dark-mode {
 .container .btn-start {
   width: 171px;
   height: 54px;
-  background-color: #fbad37;
-  color: #ffff;
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
   margin-top: 67px;
   border-radius: 20px;
   border: none;
   font-size: 25px;
 }
+@media only screen and (min-width: 1300px) {
+  .img-home {
+    display: block;
+    width: 500px;
+    margin-left: 100px;
+    margin-top: 50px;
+  }
+  .container {
+    display: flex;
+    justify-content: space-between;
```

Comment:
![image](https://user-images.githubusercontent.com/76267220/128865406-1c7d6612-5906-409a-aa7b-3b2bcacc7d46.png)

just use space-around instead of space-between to solve it 🙃